src/main/java/au/edu/wehi/idsv/AssemblyEvidenceSource.java

```diff
@@ -505,7 +506,7 @@ public static void validateAllCategoriesAssembled(ProcessingContext pc, List<Ass
 			List<String> extra = aes1.getAssemblyCategories().stream()
 					.filter(x -> !pcCategories.contains(x))
 					.collect(Collectors.toList());
-			if (extra.size() > 0) {
+			if (!extra.isEmpty() && !(pc.getCommandLineProgram() instanceof IdentifyVariants)){
```

Comment:
Can you please find clearer name than Empty? "Empty" is how we chose to implement the case when the variants are only called from the assembly, but I believe that it is not clear to the user of this code